bucketlist/settings/base.py

```diff
@@ -107,3 +109,20 @@
 # https://docs.djangoproject.com/en/1.11/howto/static-files/
 
 STATIC_URL = '/static/'
+STATICFILES_FINDERS = (
+    'django.contrib.staticfiles.finders.FileSystemFinder',
+    'django.contrib.staticfiles.finders.AppDirectoriesFinder',
+    'djangobower.finders.BowerFinder',
+)
+BOWER_COMPONENTS_ROOT = os.path.join(BASE_DIR, 'bucketlist_app/static')
```

Comment:
Good to have space here to show a new configuration set

Reply:
Done!